tests/integration/docusaurus/setup/configuring_data_docs/how_to_host_and_share_data_docs_on_gcs.py

```diff
@@ -228,12 +251,13 @@ def server_error(e):
 great_expectations docs build --site-name new_site_name
 """
 
-result = subprocess.Popen(
+with subprocess.Popen(
```

Comment:
oo this an interesting change. We needed better context management?

Reply:
I'm not sure if the test was failing due to this in CICD. I was not able to repro the test failure. I did however encounter a warning (see screenshot in description) and this resolves that. It could be the reason why it failed, it may speed up the test slightly, or at the very least use some better best practices.